debian/molior-server.postinst

```diff
@@ -33,14 +33,16 @@ case "$1" in
             useradd -m -G sbuild --shell /bin/bash --home-dir $MOLIOR_HOME $MOLIOR_USER
         fi
 
-        mkdir -p "${MOLIOR_HOME}/.ssh"
+        mkdir -p ${MOLIOR_HOME}/.ssh
```

Comment:
Why remove this?

Reply:
it is not needed. will remove the {} as well... thanks ;)

Reply:
done